### DISCUSSION TITLE
Bandwidth consumed?

### DISCUSSION THREAD

**Discussion by MarcSallent** [ant-media/Ant-Media-Server#5545] [Q&A]
Hi there! Is there a way to know how many GB has an application used? I don't want to do num_streams x num_viewers x ... but to know the exact value of bandwidth consumed

Thanks

**Comment by lastpeony** [comment]
Hello, currently, Ant Media Server does not provide this specific statistic. You can submit a feature request to contact@antmedia.io to create a mutually beneficial solution. In the meantime, you can monitor Ant Media Server usage either through your cloud service provider or by using a Linux bandwidth monitoring tool such as iftop.
